package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "20-intro_to_angular_and_webpack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "lint": "eslint *.js app/*.js",
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
```

Comment:
Although karma start will run your tests, it's best practice to include an npm script that will run your tests for you 👍